File: python/1100-1199/1110-1119/1115.py
while True:
    entry = list(map(int, input().split()))
    if entry[0] == 0 or entry[1] == 0:
        break
    if entry[0] > 0 and entry[1] > 0:
        print("primeiro")
    if entry[0] < 0 and entry[1] > 0:
        print("segundo")
    if entry[0] < 0 and entry[1] < 0:
        print("terceiro")
    if entry[0] > 0 and entry[1] < 0:
        print("quarto")

File: C99/1100-1199/1110-1119/1115.c
#include <stdio.h>

int main()
{
    int n1, n2, conf = 0;
    while (conf == 0)
    {
        scanf("%d %d", &n1, &n2);
        if (n1 != 0 && n2 != 0)
        {
            if (n1 > 0 && n2 > 0)
            {
                printf("primeiro\n");
            }
            else if (n1 > 0 && n2 < 0)
            {
                printf("quarto\n");
            }
            else if (n1 < 0 && n2 > 0)
            {
                printf("segundo\n");
            }
            else if (n1 < 0 && n2 < 0)
            {
                printf("terceiro\n");
            }
        }
        else
        {
            conf++;
        }
    }
    return 0;
}